tera-templates/core/about.html

```diff
@@ -0,0 +1,195 @@
+{% extends "base.html" -%}
+
+{%- block title -%} About Docs.rs {%- endblock title -%}
+
+{%- block body -%}
+    {%- set docsrs_repo = "https://github.com/rust-lang/docs.rs" -%}
+
+    <h1 id="crate-title">About Docs.rs</h1>
+    <div class="container about">
+        <p>
+            Docs.rs (formerly cratesfyi) is an
+            <a href="{{ docsrs_repo }}">open source</a>
+            documentation host for crates of the
+            <a href="https://www.rust-lang.org/">Rust Programming Language</a>.
+        </p>
+
+        <p>
+            Docs.rs automatically builds crates' documentation released on
+            <a href="https://crates.io/">crates.io</a>
+            using the nightly release of the Rust compiler.
+
+            {%- if rustc_version -%}
+            The current version of the Rust compiler in use is <code>{{ rustc_version }}</code>.
```

Comment:
It looks like tera cuts out all this whitespace, so it looks like `compiler.The`. Do you know how to fix that? https://docs.rs/about

Reply:
It's the `-` on the above tag, they say to suppress all whitespace on that side of the tag.

Reply:
Fixed in #840